phpBB/includes/functions.php

```diff
@@ -4279,13 +4283,14 @@ function obtain_users_online_string($online_users, $item_id = 0, $item = 'forum'
 		* @var	string	item			Restrict online users to a certain
 		*								session item, e.g. forum for
 		*								session_forum_id
-		* @var	string	sql				SQL query to obtain users online data
+		* @var	string	sql_ary				SQL query to obtain users online data
```

Comment:
you have an extra tab here => `		* @var	string	sql_ary			SQL query to obtain users online data`